recipes/openmm-atmmetaforce-plugin/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DOPENMM_DIR=$PREFIX"
+
+if [[ "$target_platform" == linux* ]]; then
+    # CFLAGS
+    MINIMAL_CFLAGS+=" -O3 -ldl"
+    CFLAGS+=" $MINIMAL_CFLAGS"
+    CXXFLAGS+=" $MINIMAL_CFLAGS"
+    LDFLAGS+=" $LDPATHFLAGS"
+
+    # Use GCC
+    CMAKE_ARGS+=" -DCMAKE_C_COMPILER=$CC -DCMAKE_CXX_COMPILER=$CXX"
+
+    # CUDA_HOME is defined by nvcc metapackage
+    CMAKE_ARGS+=" -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_HOME}"
+    CMAKE_ARGS+=" -DCMAKE_LIBRARY_PATH=${CUDA_HOME}/lib64/stubs"
+    CMAKE_ARGS+=" -DCUDA_CUDA_LIBRARY=${CUDA_HOME}/lib64/stubs/libcuda.so"
+fi
+
+mkdir build
+cd build
+cmake ${CMAKE_ARGS} ${SRC_DIR}
+make -j $CPU_COUNT
+make install PythonInstall
+
+for lib in ${PREFIX}/lib/plugins/*${SHLIB_EXT}; do
+    ln -s $lib ${PREFIX}/lib/$(basename $lib) || true
+done
```

Comment:
Since this is a python library with c extension, you shouldn't need to ship any c-headers. Please add a step to the build script which removes the files added to /include.

Reply:
Is this step absolutely necessary? The OpenMM package, of which this is a plugin, ships C++ header files. All of the OpenMM plugins on conda-forge with a python interface that I have seen ship C++ headers.

The C++ header of this plugin is very small. Removing it will not save a significant amount in terms of file size.

An additional reason to preserve C++ headers is that it would allow users to use the C++ API in their derivative codes, even though it would not have the full functionality of the python interface.

If this step is absolutely necessary, could you please provide an example of the commands to delete C++ headers from the default include directory? Or please provide an example of a package with a build script that does this. Thank you!

Reply:
Thank you for the help and explanations. I have committed the changes indicated.